rabbit_monitor.py

```diff
@@ -80,6 +85,28 @@ def healthcheck(session):
 
 
 @asyncio.coroutine
+def message_count(session, url=None):
+    logger.info('Getting message counts')
+    if url is None:
+        resp = yield from fetch(session, urls['messages'])
+    else:
+        resp = yield from fetch(session, url)
+
+    if resp.status == 200:
+        text = yield from resp.text()
```

Comment:
The [aiohttp.Response](http://aiohttp.readthedocs.io/en/stable/_modules/aiohttp/web_response.html#Response) object doesn't have a `.json` method.